common/components/controllers/CorporateHackathonController.jsx

```diff
@@ -263,7 +263,7 @@ class CorporateHackathonController extends React.PureComponent<{||}, State> {
           <div className="corporate-hackathon-carousel carousel-testimonial-root">
             <TestimonialCarousel
               category="partner-highlights"
-              interval={600000}
+              interval={null}
```

Comment:
Do we need to use interval={null} here, as well as in the TestimonialCarousel component itself? Carousel shouldn't be taking any props so I'm hoping it's safe to remove, but please test and let me know?